From fcea191a6e0eb869c62b5d1ea8a6e914a540fcfb Mon Sep 17 00:00:00 2001
From: Jae Hyun Yoo <jae.hyun.yoo@intel.com>
Date: Wed, 3 Oct 2018 10:17:58 -0700
Subject: [PATCH] arm: dts: aspeed: Swap the mac nodes numbering

This patch swaps the numbering of mac0 and mac1 to make a dedicated
nic get assigned the first ethernet device number.

Signed-off-by: Jae Hyun Yoo <jae.hyun.yoo@intel.com>
---
 arch/arm/boot/dts/aspeed-g4.dtsi | 16 ++++++++--------
 arch/arm/boot/dts/aspeed-g5.dtsi | 16 ++++++++--------
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-g4.dtsi b/arch/arm/boot/dts/aspeed-g4.dtsi
index 3df43f925848..618abeab539b 100644
--- a/arch/arm/boot/dts/aspeed-g4.dtsi
+++ b/arch/arm/boot/dts/aspeed-g4.dtsi
@@ -101,14 +101,6 @@
 			reg = <0x1e6c2000 0x80>;
 		};
 
-		mac0: ethernet@1e660000 {
-			compatible = "aspeed,ast2400-mac", "faraday,ftgmac100";
-			reg = <0x1e660000 0x180>;
-			interrupts = <2>;
-			clocks = <&syscon ASPEED_CLK_GATE_MAC1CLK>;
-			status = "disabled";
-		};
-
 		mac1: ethernet@1e680000 {
 			compatible = "aspeed,ast2400-mac", "faraday,ftgmac100";
 			reg = <0x1e680000 0x180>;
@@ -117,6 +109,14 @@
 			status = "disabled";
 		};
 
+		mac0: ethernet@1e660000 {
+			compatible = "aspeed,ast2400-mac", "faraday,ftgmac100";
+			reg = <0x1e660000 0x180>;
+			interrupts = <2>;
+			clocks = <&syscon ASPEED_CLK_GATE_MAC1CLK>;
+			status = "disabled";
+		};
+
 		ehci0: usb@1e6a1000 {
 			compatible = "aspeed,ast2400-ehci", "generic-ehci";
 			reg = <0x1e6a1000 0x100>;
diff --git a/arch/arm/boot/dts/aspeed-g5.dtsi b/arch/arm/boot/dts/aspeed-g5.dtsi
index 6e0b40eb939f..b8c85fad2a39 100644
--- a/arch/arm/boot/dts/aspeed-g5.dtsi
+++ b/arch/arm/boot/dts/aspeed-g5.dtsi
@@ -149,14 +149,6 @@
 			reg = <0x1e6c2000 0x80>;
 		};
 
-		mac0: ethernet@1e660000 {
-			compatible = "aspeed,ast2500-mac", "faraday,ftgmac100";
-			reg = <0x1e660000 0x180>;
-			interrupts = <2>;
-			clocks = <&syscon ASPEED_CLK_GATE_MAC1CLK>;
-			status = "disabled";
-		};
-
 		mac1: ethernet@1e680000 {
 			compatible = "aspeed,ast2500-mac", "faraday,ftgmac100";
 			reg = <0x1e680000 0x180>;
@@ -165,6 +157,14 @@
 			status = "disabled";
 		};
 
+		mac0: ethernet@1e660000 {
+			compatible = "aspeed,ast2500-mac", "faraday,ftgmac100";
+			reg = <0x1e660000 0x180>;
+			interrupts = <2>;
+			clocks = <&syscon ASPEED_CLK_GATE_MAC1CLK>;
+			status = "disabled";
+		};
+
 		ehci0: usb@1e6a1000 {
 			compatible = "aspeed,ast2500-ehci", "generic-ehci";
 			reg = <0x1e6a1000 0x100>;
-- 
2.7.4

